src/pages/courses/full/index.njk

```diff
@@ -11,16 +11,116 @@ nav:
+  info:
+    - title: Benefits
+      items:
+        - label: "100% free self-paced online learning"
+        - label: "Earn a Certificate to share on your résumé and social media"
+        - label: "Discuss, share, and get feedback in the course forum"
+        - label: "Optional homework and quizzes to sharpen your skills"
+    - title: Length
+      items:
+        - label: "Video Lessons: 3–5 hours"
+        - label: "Course Duration: 12–22 hours"
+  promo:
+    pitch: Don’t have the time?
+    action: Explore our Gym Shorts.
 ---
-<header>
-  <h1>{{ intro.heading }}</h1>
-  <p><strong>{{ intro.subheading }}</strong> {{ intro.content }}</p>
+<header class="intro bg-mono-300 full-bleed">
+  <hgroup>
+    <h1>{{ intro.heading }}</h1>
+    <p><b>{{ intro.subheading }}</b> {{ intro.action }}</p>
+    <!-- <p>{{ intro.content }}</p> -->
+  </hgroup>
+  <div class="list-summary">
+    {% for info in intro.info %}
+    <dl>
+      <dt>{{ info.title }}</dt>
+      {% for item in info.items %}
+      <dd>{{ item.label }}</dd>
+      {% endfor %}
+    </dl>
+    {% endfor %}
+  </div>
+  <!-- <p><b>{{ intro.promo.pitch }}</b> {{ intro.promo.action }}</p> -->
 </header>
 
-<h2>Full Courses</h2>
-{% for course in collections.live_courses_full %}
-  {%- set module = course -%}
-  {%- include 'modules/panels/course.njk' -%}
-{% endfor %}
+<div id="all-topics" class="topic-select"></div>
+<div id="featured" class="topic-select"></div>
+<div id="design" class="topic-select"></div>
+<div id="development" class="topic-select"></div>
+<div id="ux" class="topic-select"></div>
+<div id="career-skills" class="topic-select"></div>
+
+<div id="topics" class="topics">
+
+<nav class="topic-menu">
+  <h2>Explore</h2>
+  <ul>
+    <li><a href="#all-topics">All Topics</a></li>
+    <li><a href="#featured">Featured</a></li>
+  </ul>
+  <ul>
+    <li><a href="#design">Design</a></li>
+    <li><a href="#development">Development</a></li>
+    <li><a href="#ux">UX</a></li>
+    <li><a href="#career-skills">Career Skills</a></li>
+  </ul>
+</nav>
+
+<article class="topic-listing">
+  <h2 class="topic-label" aria-hidden="true"><span>All Topics</span></h2>
+  <ul class="courses">
+  {% for course in collections.live_courses_full | reverse %}
+    {%- set module = course -%}
+    {%- include 'modules/panels/course.njk' -%}
+  {% endfor %}
+
+    <!-- TODO: Update placeholder with include and custom logic -->
+
+    <li class="course gym-001 career-skills elective">
+      <p class="note">
+        <b>Suggested Gym Short:</b> While we don’t have a Full Course, we recommend this Gym Short instead.
+      </p>
+      <article id="gym-001" class="course-card">
+        <div class="main">
+          <a href="">
+            <img class="artwork" alt="Defeating Busy course artwork." src="https://gym.soy/img/courses/svg/gym-001.svg" srcset="https://gym.soy/img/courses/svg/gym-001.svg" decoding="async" fetchpriority="high" width="574" height="488">
```

Comment:
I think we should use relative URLs here.

Reply:
Agreed. Updated the copy-paste markup.